liblangutil/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Solidity Commons Library (Solidity related sharing bits between libsolidity and libyul)
+file(GLOB sources "*.cpp")
```

Comment:
I think we should start listing files explicitly. It makes it much harder to forget checking in a new file and also avoids problems with .cpp files that are accidentally kept in a directory.

Reply:
let me do that in a seperate PR, please, so I can do that globally on all CMakeList.txt's :)